integrations/cassandra/src/cassandra.go

```diff
@@ -23,7 +23,7 @@ type argumentList struct {
 
 const (
 	integrationName    = "com.newrelic.cassandra"
-	integrationVersion = "1.1.0"
+	integrationVersion = "1.2.0"
```

Comment:
Don't you have to modify also this value in the integration tests?

Reply:
Yes, but unfortunately integration tests are in a different repository. I will create the PR there for updating it.